example.cpp

```diff
@@ -0,0 +1,13 @@
+#include <boost/lambda/lambda.hpp>
```

Comment:
What is this file for?

Reply:
It was for testing and debugging, forgot to remove